docs/user/README.md

```diff
@@ -87,7 +95,7 @@ The `node-image` in turn is built off the [`base-image`][base image], which
 installs all the dependencies needed for Docker and Kubernetes to run in a
 container.
 
-See [base image](#base-image), for more advanced information information.
+See [base image](#base-image), for more advanced information.
```

Comment:
there is a separate PR that fixes this:
https://github.com/kubernetes-sigs/kind/pull/163